### PR TITLE
Add migros.ch

### PR DESCRIPTION
**Pull request by luschmar**
login.migros.ch just released MFA support.

### CODE REVIEW

File: entries/m/migros.ch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "Migros": {
+    "passwordless": "allowed",
+    "mfa": "allowed",
+    "categories": "retail",
+    "documentation": "https://account.migros.ch/help#faqPasskeyPrio",
+    "recovery": "https://account.migros.ch/help#faqPasskeyPrio",
+    "regions": [
+      "at",
```
Comment (hkamran80) [inline entries/m/migros.ch.json:9]: I don't see this region listed in the [2FA Directory PR](https://github.com/2factorauth/twofactorauth/pull/8255).

File: entries/m/migros.ch.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "Migros": {
+    "passwordless": "allowed",
+    "mfa": "allowed",
```
Comment (hkamran80) [inline entries/m/migros.ch.json:4]:
```suggestion
```

From the documentation, it does not seem like Migros supports MFA with FIDO2. Can you confirm?

File: entries/m/migros.ch.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "categories": "retail",
+    "documentation": "https://account.migros.ch/help#faqPasskeyPrio",
+    "recovery": "https://account.migros.ch/help#faqPasskeyPrio",
```
Comment (hkamran80) [inline entries/m/migros.ch.json:7]:
```suggestion
```

This is not a recovery documentation page. Per [2FA Directory's `CONTRIBUTING.md`](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#other-properties):
> The recovery field can be used to link to account recovery documentation about what to do if you lose access to your 2FA method.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/m/migros.ch.json (hunk 2 above)
Comment (luschmar) [reply entries/m/migros.ch.json:4]: On the first step, we implemented passkeys as a MFA. You can still use your old password to access your account on the SFA level. You gain your MFA status with the passkey.

Comment (hkamran80) [reply entries/m/migros.ch.json:4]: That's not noted in the documentation though.

File: entries/m/migros.ch.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
     "mfa": "allowed",
     "categories": "retail",
     "documentation": "https://account.migros.ch/help#faqPasskeyPrio",
-    "recovery": "https://account.migros.ch/help#faqPasskeyPrio",
+    "recovery": "https://account.migros.ch/help#faqPasskeyProblem",
```
Comment (Carlgo11) [inline entries/m/migros.ch.json:7]: That link doesn't work for me.
image
```
Uncaught TypeError: z.current is null
    t NextJS
_app-15c1dfebc9027a7c.js:1:1509520
```

Comment (hkamran80) [reply entries/m/migros.ch.json:7]: I'm also getting that error now

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**